pkg/proxy/iptables/proxier.go

```diff
@@ -788,11 +792,25 @@ func (proxier *Proxier) execConntrackTool(parameters ...string) error {
 // assumes proxier.mu is held
 func (proxier *Proxier) syncProxyRules() {
 	if proxier.throttle != nil {
```

Comment:
Yes, default atm is to do nothing proxier.throttle is nill without input option.